### PR TITLE
feat(scrape_harvard_ia): Add support for multiple volumes

### PR DESCRIPTION
**Pull request by flooie**
Some volumes are large, added support for large volumes
split across multiple IA directories.

### CODE REVIEW

File: cl/lib/argparse_types.py [hunk 1]
```diff
@@ -37,15 +38,17 @@ def readable_dir(prospective_dir):
...
     """
+    if ":" not in volumes_arg:
+        return [volumes_arg]
     volumes = [int(e) if e.strip() else 2000 for e in volumes_arg.split(":")]
     if len(volumes) == 1:
         start = stop = volumes[0]
     else:
         start, stop = volumes[0], volumes[1] + 1
-    return range(start, stop)
+    return [*range(start, stop)]
```
Comment (mlissner) [inline cl/lib/argparse_types.py:54]: I think it's a bit more pythonic to do:

```suggestion
    return list(range(start, stop))
```

File: cl/scrapers/management/commands/scrape_harvard_ia.py [hunk 2]
```diff
@@ -100,7 +109,8 @@ def get_from_ia(options: OptionsType) -> None:
     """
     ia_session = create_ia_session()
     volumes_ids_for_reporter = fetch_ia_volumes(ia_session, options)
-
+    print(volumes_ids_for_reporter)
+    return
```
Comment (mlissner) [inline cl/scrapers/management/commands/scrape_harvard_ia.py:113]: Realizing this must not be ready for review yet. My bad. Anyway...

**Review by mlissner — COMMENTED** [review]
I just realized you didn't ask for review yet, but I already had these two pending comments, so sending them along. Rest looks fine, FWIW. :)

File: cl/scrapers/management/commands/scrape_harvard_ia.py (hunk 2 above)
Comment (flooie) [reply cl/scrapers/management/commands/scrape_harvard_ia.py]: always happy to have your second opinion - anytime

**Comment by mlissner** [comment]
I don't completely understand the plumbing going on here, but if it works and you're happy, great.

**Comment by flooie** [comment]
Thanks.

**Pull request merged**